### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by itsnitish22**
## Guidelines
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

### Types of changes
What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Preparing a pull request for review
Ensure your change is properly formatted by running:

```gradle
$ ./gradlew spotlessApply
```

Please correct any failures before requesting a review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -525,8 +525,8 @@ balloon.clearAllPreferences()
 ```
 
 ### Avoid Memory leak
-Dialog, PopupWindow and etc, have memory leak issue if not dismissed before activity or fragment are destroyed.<br>
-But Lifecycles are now integrated with the Support Library since Architecture Components 1.0 Stable released.<br>
+Dialog, PopupWindow and etc, canhave memory leak issue if not dismissed before the activity or fragment are destroyed.<br>
```
Comment (skydoves) [inline README.md:528]:
```suggestion
Dialog, PopupWindow and etc, can have memory leak issues if not dismissed before the activity or fragment is destroyed.<br>
```

File: README.md [hunk 2]
```diff
@@ -642,7 +642,7 @@ BalloonHighlightAnimation.ROTATE
 .setBalloonHighlightAnimation(BalloonHighlightAnimation.SHAKE)
 ```
 
-We can implement the rotate animation like below the example:
+We can implement  rotate animation like the example below:
```
Comment (skydoves) [inline README.md:645]:
```suggestion
We can implement rotate animation like the example below:
```

File: README.md [hunk 3]
```diff
@@ -626,7 +626,7 @@ FADE | OVERSHOOT | ELASTIC | CIRCULAR |
 | <img src="https://user-images.githubusercontent.com/24237865/74601168-6115c580-50de-11ea-817b-a334f33b6f96.gif" align="center" width="100%"/> | <img src="https://user-images.githubusercontent.com/24237865/74601171-6410b600-50de-11ea-9ba0-5634e11f148a.gif" align="center" width="100%"/> | <img src="https://user-images.githubusercontent.com/24237865/74601170-63781f80-50de-11ea-8db4-93f1dd1291fc.gif" align="center" width="100%"/> | <img src="https://user-images.githubusercontent.com/24237865/74607359-b6bc9300-511b-11ea-978b-23bcc4399dce.gif" align="center" width="100%"/> |
 
 ### BalloonHighlightAnimation
-We can give repeated dynamic animations to the Balloon while it's showing up. The animation would work differently by the position of the arrow.
+We can give a repeated dynamic animations to the Balloon while it's showing up. The animation would work differently by the position of the arrow.
```
Comment (skydoves) [inline README.md:629]:
```suggestion
We can give repeated dynamic animations to the Balloon while it's showing up. The animation would work differently by the position of the arrow.
```

**Review by skydoves — CHANGES_REQUESTED** [review]
Hey, thanks for your contribution!
I left a couple of suggestions. :)

**Comment by itsnitish22** [comment]
Thanks!
You can review the changes now :)

**Review by skydoves — APPROVED** [review]
LGTM! 👍

**Pull request merged**